### ISSUE TITLE
Manual publication of seurat conversions

### ISSUE THREAD

**Issue by brianraymor** [chanzuckerberg/single-cell-data-portal#1448]
---

Collection: [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://cellxgene.cziscience.com/collections/0a839c4b-10d0-4d64-9272-684c49a2c8ba)
Revision: [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://cellxgene.cziscience.com/collections/0a839c4b-10d0-4d64-9272-684c49a2c8ba/private)
RDS for Dataset: [Large-scale single-cell analysis reveals critical immune characteristics of COVID-19 patients](https://www.dropbox.com/s/fux19dgbymooxui/Large_scale_single_cell_analysis_reveals_critical_immune_characteristics_of_COVID_19_patients.tar?dl=0)

---

Collection: [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5)
Revision: [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5/private)
RDS for Dataset: [Tabula Sapiens - All Cells](https://www.dropbox.com/s/de9gbb4xu5re5hz/tabula_sapiens.tar?dl=0)

**Comment by atolopko-czi** [comment]
For whomever takes this on, can you note here what measures it ultimately takes to convert the Seurat files?

**Comment by pablo-gar** [comment]
The conversion fails due to memory issues with datasets that have a high number of cells, usually more  than1 mi. There's two solutions and I use them sequentially (I first try number 1 and if that doesn't work try number 2)

1. Set up a high memory EC2 instance and try the [conversion](https://github.com/chanzuckerberg/single-cell-data-portal/blob/main/backend/corpora/dataset_processing/make_seurat.R) manually.
2. Split up the h5ad into smaller meaningful files, for instance by tissue or cell type. Perform the conversions for each individual subset and then package them into a tar file

After, an engineer has to manually upload the resulting file to S3 and update the database.

**Comment by brianraymor** [comment]
I will coordinate with Stanford. They will need to queue up the revisions for the collections above which may fail (as expected).

**Comment by pablo-gar** [comment]
I am in charge of updating the collections, indeed the datasets indicated here failed.

**Comment by brianraymor** [comment]
Thanks for the update.

>  indeed the datasets indicated here failed.

It's good that the portal is consistent.

**Comment by pablo-gar** [comment]
All datasets in those collections have been updated, as expected the ones below failed due to seurat conversion, and I have added the links to the manually curated ones.

I cannot edit the original text in this issue @brianraymor.

Collection: [A human cell atlas of fetal gene expression](https://cellxgene.cziscience.com/collections/c114c20f-1ef4-49a5-9c2e-d965787fb90c)
Revision: [A human cell atlas of fetal gene expression](https://cellxgene.cziscience.com/collections/c114c20f-1ef4-49a5-9c2e-d965787fb90c/private)
RDS for Dataset: [Survey of human embryonic development](https://www.dropbox.com/s/j7uc3vb7t2fzluh/A_human_cell_atlas_of_fetal_gene_expression.tar?dl=0)

---

Collection: [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://cellxgene.cziscience.com/collections/0a839c4b-10d0-4d64-9272-684c49a2c8ba)
Revision: [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://cellxgene.cziscience.com/collections/0a839c4b-10d0-4d64-9272-684c49a2c8ba/private)
RDS for Dataset: [Large-scale single-cell analysis reveals critical immune characteristics of COVID-19 patients](https://www.dropbox.com/s/tdp6p0mgd7celfs/Large_scale_single_cell_analysis_reveals_critical_immune_characteristics_of_COVID_19_patients.tar?dl=0)

---

Collection: [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5)
Revision: [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5/private)
RDS for Dataset: [Tabula Sapiens - All Cells](https://www.dropbox.com/s/cliujqfg3p4lfkg/tabula_sapiens.tar?dl=0)

**Comment by brianraymor** [comment]
I updated the top-level summary comment. @maesen - this is now ready for the sprint backlog.

**Comment by pablo-gar** [comment]
I  just realized that I made the conversions without adding human-readable labels to ontology terms and genes. I will update this again when is ready.

Sorry for the extra backlog.

I will update again when the conversions are done.

**Comment by pablo-gar** [comment]
Updated data:

Collection: [A human cell atlas of fetal gene expression](https://cellxgene.cziscience.com/collections/c114c20f-1ef4-49a5-9c2e-d965787fb90c)
Revision: [A human cell atlas of fetal gene expression](https://cellxgene.cziscience.com/collections/c114c20f-1ef4-49a5-9c2e-d965787fb90c/private)
RDS for Dataset: [Survey of human embryonic development](https://www.dropbox.com/s/7ohrhvp3sdh62ld/A_human_cell_atlas_of_fetal_gene_expression.tar?dl=0)

---

Collection: [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://cellxgene.cziscience.com/collections/0a839c4b-10d0-4d64-9272-684c49a2c8ba)
Revision: [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://cellxgene.cziscience.com/collections/0a839c4b-10d0-4d64-9272-684c49a2c8ba/private)
RDS for Dataset: [Large-scale single-cell analysis reveals critical immune characteristics of COVID-19 patients](https://www.dropbox.com/s/fux19dgbymooxui/Large_scale_single_cell_analysis_reveals_critical_immune_characteristics_of_COVID_19_patients.tar?dl=0)

---

Collection: [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5)
Revision: [Tabula Sapiens](https://cellxgene.cziscience.com/collections/e5f58829-1a66-40b5-a624-9046778e74f5/private)
RDS for Dataset: [Tabula Sapiens - All Cells](https://www.dropbox.com/s/de9gbb4xu5re5hz/tabula_sapiens.tar?dl=0)

**Comment by brianraymor** [comment]
Deleted some of the outdated comments and updated top-level summary comment to include latest links.

**Comment by MDunitz** [comment]
10/4 Updates
Fetal gene expression 
- The pipeline is timing out while running the cxg conversion. @ebezzi, @leslieypark and I added ssh access to the instance (to help with debugging) and increased the timeout (to potentially solve the problem).
Covid 19/Tabula Sapiens
- The processing pipeline is failing before completing the loom or seurat conversion. The loom format is deprecated so we did not recreate the loom files. @pablo-gar manually converted the files to the rds format and @leslieypark transferred them to the prod environment, added a dataset artifact, and updated the dataset_procesing_status.
